src/workloads/query/LargeProjections.yml

```diff
@@ -0,0 +1,204 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This workload runs collscan queries with a large projection on around 20 fields.
+
+Keywords:
+- Loader
+- CrudActor
+- QuiesceActor
+- insert
+- find
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
```

Comment:
I think we should add a small/medium version of this workload as well. Seeing when a regression only affects a larger workload vs when it only affects a smaller workload (or how much it affects each collection size) can give you a hint about whether its execution or optimization time that is problematic.